src/main/database/driveFoldersDB.ts

```diff
@@ -0,0 +1,110 @@
+import { DatabaseAdapter } from './adapters/base';
+import { RemoteSyncedFolder } from '../remote-sync/helpers';
+import SQLite from 'better-sqlite3';
+
+export function generateCreateTableQuery(): string {
+  const query = `CREATE TABLE IF NOT EXISTS drive_folders (
+    type TEXT NOT NULL,
+    id INTEGER UNIQUE NOT NULL,
+    parentId INTEGER,
+    bucket TEXT,
+    userId INTEGER, 
+    createdAt TEXT,
+    updatedAt TEXT NOT NULL,
+    uuid TEXT NOT NULL,
+    plainName TEXT,
+    status TEXT NOT NULL,
```

Comment:
@sg-gs This will fail right now as the API is not returning a folder status yet

Reply:
Still working on it. Could we set it now, using the fields deleted and removed? I am still applying this change to files table which is 20x bigger than the folders one, so it will last a bit more to finish.